cips/cip-26.md

```diff
@@ -0,0 +1,64 @@
+| cip | 26 |
+| - | - |
+| title | Reduce block time to six seconds |
+| description | Proposal to reduce block time on celestia-app to 6 seconds, from 12. |
```

Comment:
I'd rename the CIP to "versioned timeout", while the end goal is to reduce block time, but the actual change is that timeouts are now application version dependant, which would have a long-term effect from v3 on, and that is the timeouts will always be controlled by the application version (unless we change  the implementation in the future), and the timeouts in the configs won't be effective anymore.

Reply:
Sure, done!